### ISSUE TITLE
Bash tab completion causes a locale error

### ISSUE THREAD

**Issue by georgebrock** [ejholmes/vagrant-heroku#9]
Whenever I create a new instance using this base box and SSH into the instance, bash tab completion gives me this error:

```
bash: warning: setlocale: LC_CTYPE: cannot change locale (UTF-8)
```

Adding the following to `/etc/default/locale` fixes it:

```
LC_ALL="en_US.UTF-8"
```

I'm happy to submit a pull request for this, but my Linux locales knowledge is pretty sketchy so there may be a better solution.

**Comment by ejholmes** [comment]
Can't say that I'm anymore familiar with it, but this change seems like it would be fairly harmless. Feel free to open a PR.

**Comment by georgebrock** [comment]
Fixed by #14.
